packages/cli/src/commands/logs/index.ts

```diff
@@ -39,8 +39,9 @@ export default async function logs(client: Client) {
     return 1;
   }
 
-  for (const flag of deprecatedFlags) {
-    if (parsedArguments.flags[flag]) {
+  // TODO: This behavior should be centralized in `parseArguments`
```

Comment:
**comment (non-blocking)**: I like this idea, it goes well with how I want to re-model how we handle deprecation, type-wise.